src/cedar.js

```diff
@@ -482,19 +482,20 @@ Cedar.prototype._renderSpec = function(spec){
     }
     //use vega to parse the spec
     //it will handle the spec as an object or url
-    vg.parse.spec(spec, function(chartCtor) {
+    vg.parse.spec(spec, function(error, chartCtor) {
+
+      if (error) { return error; }
 
       //create the view
       self._view = chartCtor({
         el: self._elementId,
         renderer: self._renderer
-      });
-
+      }).update();
 
       var width = self.width || parseInt(d3.select(self._elementId).style('width'), 10) || 500;
       var height = self.height || parseInt(d3.select(self._elementId).style('height'), 10) || 500;
```

Comment:
@tomwayson Currently height/width in Cedar is determined by: 1) if we pass anything into it when we call `chart.show()` OR 2) The height / width of the DOM element (which seems to inherit from the parent dom element) OR 3) 500 for each. 

Do we want to allow the second option?